### ISSUE TITLE
Bug: The search bar cannot be filled with `getSelection()`

### ISSUE THREAD

**Issue by dnknn** [ssborbis/ContextSearch-web-ext#361]
Chrome86、Win10、ContextSearch version: 59b719c30e2bc86f3e6275b696043a45f60f1c08

- The search bar cannot be filled with `getSelection().toString().trim()`.
 	*  ❌ has problem : [bug1](https://user-images.githubusercontent.com/10995302/125716049-9c6d26e3-cc0c-4236-8318-6ec6850f7e3b.gif)	[image](https://user-images.githubusercontent.com/10995302/125716173-78d891e0-5f4f-46f4-a532-898ff664c80f.gif)
 	 	* `body[data-menu=sidebar] #searchBar`
 	 	* `body[data-menu=searchbar] #searchBar`

```js
if(getSelection().toString().trim()===true)	{
									// ❌ Current behavior:
	document.getElementById(`searchBar`).value = lastUsed;
}
```

```js
if(getSelection().toString().trim()===true)	{
									// ✔️ Expected behavior:
	document.getElementById(`searchBar`).value = getSelection().toString().trim();
}
```

 - - ✔️ no problem : 
 	 	* `body[data-menu=quickmenu] #searchBar`
 	 	* ContextMenu
 	 	* no problem with the 1.29 version.
 	 	
 	　
 > When I upgraded from version 1.29 to the latest version, I ran into this problem first. I don't know why it happened. And I checked all the options later, but still couldn't solve the problem.

> In addition, I feel that the latest version seems to have a lot of problems, and it seems that it is not very stable.

**Comment by ssborbis** [comment]
I think the bulk of the instability is a bad config upgrade. I'll need to get your config to see what's going on.

**Comment by dnknn** [comment]
> I think the bulk of the instability is a bad config upgrade. I'll need to get your config to see what's going on.

No, no, I used brand new data to test, and the result was the same.

Because I had the same idea as you at the beginning, I thought it was the configuration data that polluted/interfered with the normal operation of the latest version.

So I uninstalled completely, restarted the browser, and then installed a new test (initialized, not imported my own configuration), the result is the same.
